tests/extensions_tests/evaluator_tests/test_detection_coco_evaluator.py

```diff
@@ -104,4 +107,75 @@ def test_current_report(self):
         self.assertEqual(reporter.observation, mean)
 
 
+@unittest.skipUnless(_available, 'pycocotools is not installed')
+class TestDetectionCOCOEvaluatorMPI(unittest.TestCase):
+
+    def setUp(self):
+        self.comm = create_communicator('naive')
+
+        batchsize_per_process = 5
+        batchsize = batchsize_per_process * self.comm.size
+        if self.comm.rank == 0:
+            bboxes = [generate_random_bbox(5, (256, 324), 24, 120)
+                      for _ in range(10)]
+            labels = [np.random.choice(np.arange(3, dtype=np.int32), size=(5,))
+                      for _ in range(10)]
+        else:
+            bboxes = None
+            labels = None
+        initial_count = self.comm.rank * batchsize_per_process
+
+        bboxes = self.comm.bcast_obj(bboxes)
+        labels = self.comm.bcast_obj(labels)
+        self.bboxes = bboxes
+        self.labels = labels
+
+        areas = [[np.array([(bb[2] - bb[0]) * bb[3] - bb[0]])
+                  for bb in bbox] for bbox in bboxes]
+        crowdeds = [np.zeros((5,)) for _ in range(10)]
+        self.dataset = TupleDataset(
+            np.random.uniform(size=(10, 3, 32, 48)),
+            bboxes, labels, areas, crowdeds)
+        self.initial_count = initial_count
+        self.batchsize = batchsize
+
+    @attr.mpi
```

Comment:
You can put `@attr.mpi` on the class.
https://github.com/chainer/chainercv/blob/master/tests/links_tests/model_tests/ssd_tests/test_multibox_loss.py#L126-L127